cluster/packages.sls

```diff
@@ -17,6 +17,15 @@ patterns-ha-ha_sles:
         attempts: 3
         interval: 15
 
+# FIXME! This should be in the patters-ha-ha_sles
```

Comment:
Regarding FIXME , my personal view on this:

generally I'm don't have a strong opinion on introducing them or not using them.

But if we use them we should refer to a link or issue which track the upstream blocker.


like  in our case something 

`# FIXME! This should be in the patters-ha-ha_sles #jsc-139123` which refers to JIRA whatever

Reply:
I've made installing the `corosync-qdevice` conditional.